client/src/main/java/tds/assessment/Item.java

```diff
@@ -24,7 +24,7 @@
     private boolean fieldTest;
     private boolean required;
     private String strand;
-    private List<ItemProperty> itemProperties;
+    private List<ItemProperty> itemProperties = new ArrayList<>();
```

Comment:
If we want true immutability this could be `final`, the getter could return an ImmutableList.copyOf(...) and the setter could clear and add all...